### DISCUSSION TITLE
How to measure noise levels with Wio Terminal | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1787] [Q&A]
# How to measure noise levels with Wio Terminal | Seeed Studio Wiki

How to measure noise levels with Wio Terminal

https://wiki.seeedstudio.com/Wio-Terminal-Sound-Meter/

**Comment by usermctech** [comment]
Why not use the onboard microphone?
